### DISCUSSION TITLE
proposal to add a category / login list

### DISCUSSION THREAD

**Discussion by nactrimo** [dani-garcia/vaultwarden#3414] [Ideas]
I would like to propose the addition of a category such as identifier or list with prerecorded logins next to the button generated logins, we could add some login reccurent such as our mails and our nicknames

**Comment by BlackDex** [comment]
I would suggest to make these suggestions at the Bitwarden Feature Requests Forum, as this is a client related item.
https://community.bitwarden.com/c/feature-requests/5
